ui/src/app/shared/flo/support/properties-group-model.ts

```diff
@@ -42,9 +41,9 @@ export class PropertiesGroupModel extends Properties.PropertiesGroupModel {
             }
           } else if (Array.isArray(property.metadata.options)) {
             return new Properties.SelectControlModel(property,
-              Properties.InputType.SELECT, (<Array<string>> property.metadata.options).map(o => {
+              Properties.InputType.SELECT, (<Array<string>> property.metadata.options).filter(o => o.length > 0).map(o => {
```

Comment:
I will merge BUT this feels like code-smell. We really need to have proper Java-style enum-support across the app. I wish Typescript was a bit better in that regard. Maybe we should consider using `ts-enums` --> https://github.com/LMFinney/ts-enums